gravitee-apim-e2e/ui-test/integration/apim/ui/apis/ui-api-plans.spec.ts

```diff
@@ -259,25 +286,34 @@ describe('API Plans Feature', () => {
     cy.getByDataTestId('api_plans_nextstep').click();
     cy.get('[type="submit"]').contains('Create').click();
     cy.contains('Configuration successfully saved!').should('be.visible');
+
+    // publish plan
     cy.get('[type="button"]').contains('STAGING').click();
     cy.url().should('include', '/plans?status=STAGING');
     cy.contains(`${planName}-Keyless`).should('be.visible');
-    cy.getByDataTestId('api_plans_publish_plan_button').first().click();
+
+    // cy.getByDataTestId('api_plans_publish_plan_button').first().click();
```

Comment:
can you remove this line @ctschacher

Reply:
Oh, I somehow didn't see this comment of you. I'll include the change in one of the next changes. Thanks!